util/util.go

```diff
@@ -473,6 +490,19 @@ func PromptForPassword() (string, error) {
 	return strings.TrimSpace(password), nil
 }
 
+func PromptForPasswordWithLabel(label string) (string, error) {
```

Comment:
We should just use these varieties in the other parts of the code. Choose a sane label for those, or just pass empty string if there is no obvious label.